frontend/src/container/InfraMonitoringK8s/utils.tsx

```diff
@@ -14,6 +14,13 @@ import {
 } from 'components/QuickFilters/QuickFilters';
 import { DataTypes } from 'types/api/queryBuilder/queryAutocompleteResponse';
 
+export interface IEntityColumn {
```

Comment:
Using this instead of `IPodColumn`. As we're reusing it, thought it should have a more generic name.
We can remove `IPodColumn` in the end. I didn't rename it now to avoid conflicts.